var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -126,7 +127,7 @@ class Abinit(AutotoolsPackage):
     )
 
     patch("rm_march_settings.patch", when="@:8")
-    patch("rm_march_settings_v9.patch", when="@9:")
+    patch("rm_march_settings_v9.patch", when="@9:9.99")
```

Comment:
You don't need to do use the ending range value if you just use `when="@9"`, which will mean all `9.x.y` (see docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges).

Reply:
Thanks!